bridge/slack/handlers.go

```diff
@@ -197,15 +197,15 @@ func (b *Bslack) handleMessageEvent(ev *slack.MessageEvent) (*config.Message, er
 func (b *Bslack) handleStatusEvent(ev *slack.MessageEvent, rmsg *config.Message) bool {
 	switch ev.SubType {
 	case sChannelJoined, sMemberJoined:
-		b.populateUsers()
+		b.populateUsers(false)
 		// There's no further processing needed on channel events
 		// so we return 'true'.
 		return true
 	case sChannelJoin, sChannelLeave:
 		rmsg.Username = sSystemUser
 		rmsg.Event = config.EventJoinLeave
 	case sChannelTopic, sChannelPurpose:
-		b.populateChannels()
+		b.populateChannels(false)
```

Comment:
Maybe not as part of this PR but I can't seem to remember why we are calling populateChannels here and don't just register the new info from the event in the corresponding channel.

Reply:
it's added in f5659d45
let's keep this for another PR yes